WTWSMS/events/VIET_Jewish_Flavor_Events.txt

```diff
@@ -62,8 +65,8 @@ character_event = {
 	option = { #I hate them too
 		name = "EVTOPTB1000109021"
 		random_list = {
-			90 = {}
-			10 = {
+			95 = {}
+			5 = {
```

Comment:
I think they are not. Also, I had to change a lot of these to make the Ebionites and such make sense at all.